src/datadoc/frontend/fields/display_dataset.py

```diff
@@ -103,14 +103,14 @@ class DatasetIdentifiers(str, Enum):
     DatasetIdentifiers.SHORT_NAME: MetadataInputField(
         identifier=DatasetIdentifiers.SHORT_NAME.value,
         display_name="Kortnavn",
-        description="Navn på (fysisk) datafil, datatabell eller datasett",
+        description="Datasettes tekniske kortnavn (uten versjonsnummer og filendelse).",
         obligatory=True,
         editable=False,
     ),
     DatasetIdentifiers.ASSESSMENT: MetadataDropdownField(
         identifier=DatasetIdentifiers.ASSESSMENT.value,
         display_name="Verdivurdering",
-        description="Verdivurdering (sensitivitetsklassifisering) for datasettet.",
+        description="Verdivurderingen utledes fra datatilstanden og kan ha verdiene sensitiv (kildedata), skjermet (inndata, klargjorte data og statistikk) og åpen (utdata).",
```

Comment:
I think it should be colon before the list of values. "...kan ha verdiene: sensitiv (kildedata), skjermet ...."